### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by arty987**

### CODE REVIEW

File: src/components/atoms/__tests__/AppInput.spec.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import {shallowMount} from '@vue/test-utils'
+import AppInput from '@/components/atoms/AppInput.vue';
+
+describe('AppInput.vue', () => {
+    it('Should render AppInput', () => {
+        const wrapper = shallowMount(AppInput, {
```
Comment (KHarpushkina) [inline src/components/atoms/__tests__/AppInput.spec.js]: You don't need to call a shallowMount for each test, just create a const at the top of the file and then use it in all the 'it' blocks

File: src/components/atoms/__tests__/AppCheckbox.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import AppCheckbox from '@/components/atoms/AppCheckbox.vue';
+
+describe('AppCheckbox.vue', () => {
+    it('Should render AppCheckbox with default props', () => {
+        const wrapper = shallowMount(AppCheckbox);
+        expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('Should render AppCheckbox with props set', () => {
+        const wrapper = shallowMount(AppCheckbox, {
```
Comment (KHarpushkina) [inline src/components/atoms/__tests__/AppCheckbox.spec.js]: You can use setProps to change the properties of your component, no need to call shallowMount again

**Review by KHarpushkina — CHANGES_REQUESTED** [review]
The tests look great 👍

File: src/components/atoms/__tests__/AppInput.spec.js (hunk 1 above)
Comment (arty987) [reply src/components/atoms/__tests__/AppInput.spec.js]: Replaced multiple shallowMount calls with the constant

File: src/components/atoms/__tests__/AppCheckbox.spec.js (hunk 2 above)
Comment (arty987) [reply src/components/atoms/__tests__/AppCheckbox.spec.js]: Replaced multiple shallowMount calls with the constant

**Review by KHarpushkina — APPROVED** [review]

**Pull request merged**